feature-libs/quote/components/details/comment/quote-details-comment.component.html

```diff
@@ -1,20 +1,25 @@
 <ng-container *ngIf="quoteDetails$ | async as quoteDetails">
-  <div class="quote-comment-toggle" (click)="showComments = !showComments">
-    <cx-icon
-      aria-hidden="false"
-      [type]="showComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
+  <ng-container *ngIf="showComments(quoteDetails)">
+    <div
+      class="quote-comment-toggle"
+      (click)="expandComments = !expandComments"
     >
-    </cx-icon>
-    <span class="quote-comment-text">{{
-      'quote.comments.title' | cxTranslate
-    }}</span>
-  </div>
-  <div *ngIf="showComments">
-    <cx-messaging
-      [messageEvents$]="messageEvents$"
-      [messagingConfigs]="messagingConfigs"
-      (send)="onSend($event, quoteDetails.code)"
-      (itemClicked)="onItemClicked($event)"
-    ></cx-messaging>
-  </div>
+      <cx-icon
+        aria-hidden="false"
+        [type]="expandComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
+      >
+      </cx-icon>
+      <span class="quote-comment-text">{{
```

Comment:
Please, add suffix **cx**- to the class name.
It is not required to add every time **quote** to the class name because the class is used in the quote component.
It makes the class name unnecessarily longer.
You could simply define `cx-text` class